docs/book/src/topics/azure-cni-v1.md

```diff
@@ -0,0 +1,138 @@
+# Using Azure CNI V1
+
+This document provides step to use Azure CNI as the CNI solution in your workload cluster.
+
+## Azure Container Networking Interface v1
+
+While following the [quick start steps in Cluster API book](https://cluster-api.sigs.k8s.io/user/quick-start.html#quick-start), Azure CNI can be used in place of Calico as a container networking interface solution for your workload cluster.
+
+Artifacts required for Azure CNI:
+
+- [azure-cni.yaml](https://raw.githubusercontent.com/Azure/azure-container-networking/v1.5.3/hack/manifests/cni-installer-v1.yaml)
+
+<!-- TODO: will we be exposing azure cni v1 customized cluster manifest for the users to use? If no, I will put in a example manifest here for the users. -->
```

Comment:
is this still TODO?

Reply:
Good catch! I have addressed this, and isn't a TODO. https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/templates/addons/azure-cni-v1/kustomization.yaml should fetch azure-cni-v1 installer from the azure container networking repo every time users uses `make generate`